src/code/z_eff_blure.c

```diff
@@ -12,8 +12,8 @@ void EffectBlure_AddVertex(EffectBlure* this, Vec3f* p1, Vec3f* p2) {
     if (this != NULL) {
         numElements = this->numElements;
         if (numElements >= 16) {
-            // "Blure vertex addition processing: Table over %d"
-            PRINTF("ブラ─頂点追加処理:テーブルオーバー %d\n", numElements);
+            PRINTF(T("ブラ─頂点追加処理:テーブルオーバー %d\n", "Blure vertex addition processing: Table over %d\n"),
+                   numElements);
```

Comment:
This is an interesting case to consider: "bura--" is very clearly meant to mean "blur", but we call it "blure" everywhere in the codebase because of the file name. Is it worth sticking with the misspelling of the word because we do it everywhere else? I lean toward no but dont really care too much i guess

Reply:
I changed it to "blur"